appendices/VK_EXT_physical_device_drm.txt

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 Simon Ser
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+include::{generated}/meta/{refprefix}VK_EXT_physical_device_drm.txt[]
+
+=== Other Extension Metadata
+
+*Last Modified Date*::
+    2020-10-12
+*IP Status*::
+    No known IP claims.
+*Contributors*::
+    - Simon Ser
+
+=== Description
+
+This extension provides new facilities to query DRM properties for physical
+devices, enabling users to match Vulkan physical devices with DRM nodes on
+Linux.
+
+Its functionality closely overlaps with
+`EGL_EXT_device_drm`^<<VK_EXT_physical_device_drm-fn1,1>>^.
+Unlike the EGL extension, this extension does not expose a string containing
+the name of the device file and instead exposes device minor numbers.
+
+DRM defines multiple device node types. Each physical device may have one
+primary node and one render node associated. Physical devices may have no
+primary node (e.g. if the device doesn't have a display subsystem) or may have
+no render node (e.g. if it's a software rendering engine).
```

Comment:
Might be worth making it more explicit that devices may have neither (E.g., a SW rendering engine with no display system, or the NVIDIA driver if the user opts not to load our optional DRM support module).

Reply:
Done!